ios/FluentUI/Popup Menu/PopupMenuItemCell.swift

```diff
@@ -88,32 +88,12 @@ class PopupMenuItemCell: TableViewCell, PopupMenuItemTemplateCell {
         contentView.addSubview(accessoryImageView)
 
         isAccessibilityElement = true
-
-        NotificationCenter.default.addObserver(self,
-                                               selector: #selector(themeDidChange),
-                                               name: .didChangeTheme,
-                                               object: nil)
-
-        tokenSetSink = tokenSet.sinkChanges { [weak self] in
-            guard let strongSelf = self else {
-                return
-            }
-            strongSelf.updateAppearance()
-            strongSelf.updateColors()        // until popupmenuitemcell actually supports token system, clients will override colors via cell's backgroundColor property
-        }
-    }
-
-    override func willMove(toWindow newWindow: UIWindow?) {
-        super.willMove(toWindow: newWindow)
-        updateColors()
     }
 
-    @objc override func themeDidChange(_ notification: Notification) {
-        super.themeDidChange(notification)
-        guard let window = window, window.isEqual(notification.object) else {
-            return
-        }
+    override func updateAppearance() {
+        super.updateAppearance()
         backgroundStyleType = .custom
+        updateColors()
```

Comment:
Did you makes sure that the Popup Menu didn't visually change in dark mode?

Reply:
Looks fine to me, compared to a `main` build:
![Simulator Screen Shot - iPhone 14 Pro - 2023-03-08 at 18 29 17](https://user-images.githubusercontent.com/4934719/223900555-b6192dc9-4c6b-4a65-a720-7ad296cdcba2.png)